mfhdf/ncgen/ncgenyy.c

```diff
@@ -649,8 +649,8 @@ void expand_escapes();
 #include <ctype.h>
 #include <limits.h>
 #include "ncgentab.h"
-#line 653 "mfhdf/ncgen//ncgenyy.c"
-#line 654 "mfhdf/ncgen//ncgenyy.c"
+#line 652 "./mfhdf/ncgen/ncgenyy.c"
+#line 653 "./mfhdf/ncgen/ncgenyy.c"
```

Comment:
This needs the dot?

Reply:
I didn't do that on purpose - maybe the parser did it.